lib/media/presentation_timeline.js

```diff
@@ -566,3 +607,21 @@ shaka.media.PresentationTimeline = class {
     }
   }
 };
+
+/**
+ * @typedef {{
+ *   start: number,
+ *   end: number
+ * }}
+ * @export
+ *
+ * @description
+ * Defines a time range of a media segment.  Times are in seconds.
+ * Times may be relative
+ *
+ * @property {number} start
+ *   The start time of the range.
+ * @property {number} end
+ *   The end time (exclusive) of the range.
+ */
+shaka.media.PresentationTimeRange;
```

Comment:
Not needed, so please drop in favor of the type you're actually using, `shaka.dash.MpdUtils.TimeRange`

Reply:
My concern was pulling dash stuff into this area of the code that's not DASH specific